hasweb/templates/layout.html.jinja2

```diff
@@ -57,9 +57,9 @@
     </div>
     <div class="col-md-3">
       <p id="footer-twitter">
-        <a href="https://twitter.com/hasgeek" class="twitter-follow-button">Follow @hasgeek</a>
+        <a href="https://twitter.com/hasgeek" class="twitter-follow-button" target="_blank">Follow @hasgeek</a>
       </p>
-      <div class="fb-follow" data-href="https://www.facebook.com/HasGeek" data-layout="standard" data-show-faces="true">Follow on Facebook</div>
+      <div class="fb-like" data-href="https://www.facebook.com/HasGeek" data-layout="standard" data-action="like" data-size="small" data-show-faces="true" data-share="true"></div>
```

Comment:
The div needs some text (and a link) in case the Facebook iframe is blocked by an ad blocker.

Reply:
Fixed